Sources/PreviewGallery/PreviewData.swift

```diff
@@ -8,22 +8,45 @@
 import Foundation
 import SnapshotPreviewsCore
 
+
+/// A structure that manages a collection of preview types.
+///
+/// `PreviewData` is used to store and manipulate a set of `PreviewType` instances,
+/// providing functionality to filter and sort previews by module.
 public struct PreviewData {
-  let previews: [PreviewType]
+    /// The collection of preview types managed by this instance.
+    let previews: [PreviewType]
```

Comment:
What's going on with all the indentation here? can we keep it the same level to be consistent?